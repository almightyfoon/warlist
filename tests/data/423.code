c1501b_-0n8U9Hc09Aakanaj9ce59H___-0ngWf59b9b9b9b9b9b9biPiPakazan