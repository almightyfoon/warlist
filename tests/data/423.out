https://conflictchamber.com/#c1501b_-0n8U9Hc09Aakanaj9ce59H___-0ngWf59b9b9b9b9b9b9biPiPakazan

Cygnar Army - 75 / 75 points
[Theme] Heavy Metal

[Haley 2] Major Victoria Haley [+25]
 - Stormclad [18]
 - Stormwall [39]
 - Thorn [13]
 - Squire [0(5)]
Captain Arlan Strangewayes [0(4)]
Journeyman Warcaster [0(4)]
 - Firefly [8]
Lieutenant Allison Jakes [4]
 - Stormclad [18]

Cygnar Army - 73 / 75 points
[Theme] Heavy Metal

[Caine 3] Caine's Hellslingers [+20]
 - Ace [10]
 - Charger [9]
 - Charger [9]
 - Charger [9]
 - Charger [9]
 - Charger [9]
 - Charger [9]
 - Charger [9]
 - Hunter [10]
 - Hunter [10]
 - Squire [0(5)]
Archduke Alain Runewood, Lord of Fharin [0(4)]
Captain Arlan Strangewayes [0(4)]
